sql-native/src/main/java/org/ballerinalang/sql/parameterprocessor/DefaultResultParameterProcessor.java

```diff
@@ -573,7 +590,8 @@ public Object convertTimeStamp(java.util.Date timestamp, int sqlType, Type type)
                                 .TIME_OF_DAY_RECORD_SECOND), ValueCreator.createDecimalValue(second));
                         return civilMap;
                     } else {
-                        return Utils.createTimeStruct(timestamp.getTime());
+                        throw new ApplicationError("Unsupported Ballerina type:" +
+                            type.getName() + " for SQL Timestamp Data type.");
```

Comment:
```suggestion
                            type.getName() + " for SQL Timestamp data type.");
```

Reply:
Updated. Commit id is ba7d30e7b37789ba67ba680c36f81bf3a61a4588